lru/lru.go

```diff
@@ -106,8 +128,13 @@ func (c *Cache) Get(key Key) (value interface{}, ok bool) {
 		return
 	}
 	if ele, hit := c.cache[key]; hit {
+		kv := ele.Value.(*entry)
+		if kv.Expired() {
+			c.removeElement(ele)
```

Comment:
OK, I suppose consolidation of all deletes to 1 place makes it easier to follow.

TTL is time in cache, not time since last request. Right now I need the former. If the latter is ever needed, it could easily be added as a separate option.